script.module.resolveurl/lib/resolveurl/plugins/veev.py

```diff
@@ -27,8 +27,8 @@
 
 class VeevResolver(ResolveUrl):
     name = 'Veev'
-    domains = ['veev.to']
-    pattern = r'(?://|\.)(veev\.to)/(?:e|d)/([0-9a-zA-Z]+)'
+    domains = ['veev.to', 'kinoger.pw']
+    pattern = r'(?://|\.)(veev|kinoger\.to|pw)/(?:e|d)/([0-9a-zA-Z]+)'
```

Comment:
This pattern wont match the host correctly
it will match `pw` as the hostname for kinoger embeds and veev embeds wont match anything and will be broken